apps/wallet/src/ui/app/components/navigation/index.tsx

```diff
@@ -34,7 +35,12 @@ function Navigation({ className }: NavigationProps) {
                     <Tokens32 className="w-8 h-8" />
                     <span className={st.title}>Coins</span>
                 </NavLink>
-                <NavLink to="./nfts" className={makeLinkCls} title="NFTs">
+                <NavLink
```

Comment:
this is for demonstration purposes - I'll remove this and the other event and `pnpm pull-amplitude` before landing the PR